osd_planning/gcp-ccs.adoc

```diff
@@ -7,9 +7,7 @@ include::_attributes/attributes-openshift-dedicated.adoc[]
 toc::[]
 
 [role="_abstract"]
-Red Hat recommends the usage of a {GCP} project, managed by the customer, to organize all of your GCP resources. A project consists of a set of users and APIs, as well as billing, authentication, and monitoring settings for those APIs.
-
-It is a best practice for the {product-title} CCS cluster to be hosted in a GCP project within a GCP organization. The Organization resource is the root node of the GCP resource hierarchy and all resources that belong to an organization are grouped under the organization node. An IAM service account with certain roles granted is created and applied to the GCP project. When you make calls to the API, you typically provide service account keys for authentication. Each service account is owned by a specific project, but service accounts can be provided roles to access resources for other projects.
+{product-title} provides a Customer Cloud Subscription (CCS) model that allows Red Hat to deploy and manage clusters into a customer’s existing Google Cloud Platform (GCP) account.
```

Comment:
Is `into` the correct preposition here? Alternatives could be `on a customer's existing GCP account` or `in a customer's GCP account`.